dex/src/main/scala/com/wavesplatform/dex/api/websockets/WsAddressState.scala

```diff
@@ -0,0 +1,37 @@
+package com.wavesplatform.dex.api.websockets
+
+import com.wavesplatform.dex.domain.asset.{Asset, AssetPair}
+import play.api.libs.json._
+
+case class WsAddressState(balances: Map[Asset, WsBalances])
+
+object WsAddressState {
+
+  implicit val balancesMapReads: Reads[Map[Asset, WsBalances]] = Reads {
+    case JsObject(ab) => JsSuccess(ab.toMap.map { case (a, b) => AssetPair.extractAsset(a).get -> WsBalances.reads.reads(b).get })
+    case _            => JsError("Can't parse asset balances map!")
+  }
```

Comment:
The fun thing here, I have implemented a generic implicit for such cases :D Could you add a todo here:
```
// TODO: Replace with assetPairMapFormat
```
?

Reply:
Done